locales/de/pdk.po

```diff
@@ -1,604 +0,0 @@
-# German translations for puppet development kit package.
```

Comment:
I believe we ran into two error messages, one involves fuzzy messages and the other is for old messages it leaves in comments at the bottom.

Commented
```
Warning: obsolete msgid exists.
         #~ msgstr "Verwende ein eigenes Vorlagerepository für dieses Modul."
```

Fuzzy
```
Warning: fuzzy message was used.
  : msgid 'Unable to write '%{file}': %{msg}'
```